.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Backend Continuous Integration Pipeline
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: [main]
+    types: [opened, synchronize]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+        #   cache: 'npm' Is this necessary?
+      - name: enable corepack
+        run: corepack enable
+      - name: Installing Dependencies
+        run: pnpm install # pnpm does not have ci command
```

Comment:
pnpm has docs about github actions and docker.
- https://pnpm.io/continuous-integration#github-actions
- https://pnpm.io/docker

Reply:
should be good now